src/plugins/data/public/ui/query_string_input/language_switcher.tsx

```diff
@@ -52,15 +63,21 @@ export function QueryLanguageSwitcher(props: Props) {
       className="euiFormControlLayout__append kqlQueryBar__languageSwitcherButton"
       data-test-subj={'switchQueryLanguageButton'}
     >
-      {props.language === 'lucene' ? luceneLabel : kqlLabel}
+      {language === 'kuery' ? (
+        kqlLabel
+      ) : nonKqlMode === 'lucene' ? (
+        luceneLabel
+      ) : (
+        <EuiIcon type={'boxesVertical'} />
```

Comment:
I think in case there is only an icon, then the button needs a `title` attr

Reply:
done in [`e8ec7a1` (#91814)](https://github.com/elastic/kibana/pull/91814/commits/e8ec7a11a7034af5bd939913088fe2aacc874e7c)